lib/puma/launcher.rb

```diff
@@ -184,7 +185,9 @@ def run
       when :exit
         # nothing
       end
-      @binder.close_unix_paths
+        @binder.close
+        @binder.close_unix_paths
```

Comment:
I just realized - do we really want to close all sockets when restarting? Isn't that exactly the opposite of what we're supposed to do?

Reply:
I don't think the code ever reaches here in restarts.  It hits the Kernel.exec statements.